src/Mindee/Input/MindeeImageUtils.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Runtime.InteropServices;
+using Docnet.Core.Models;
+using Mindee.Exceptions;
+using SkiaSharp;
+
+namespace Mindee.Input
+{
+    /// <summary>
+    /// Utility class.
+    /// </summary>
+    public static class MindeeImageUtils
```

Comment:
Not necessarily on input, I would see this as generic Mindee image utilities.

Was not very clear first time around ...

Maybe a `Mindee.Image` namespace, like we have for Pdf?

Reply:
Ah sorry I just misunderstood, yeah that works